### ISSUE TITLE
Bug: Broker subscribe handler falls in Error when try to handle message without value (but key exists!!)

### ISSUE THREAD

**Issue by Korifey02** [airtai/faststream#1967]
python: 3.12.7
faststream: 0.5.30 and 0.5.32

Source code samle:
```py
from faststream.kafka import KafkaBroker
from faststream.kafka import KafkaMessage

broker = KafkaBroker(
    bootstrap_servers=KAFKA_SERVER,
    security=get_security(),
    log_level=logging.ERROR
)

@broker.subscriber(
    TOPIC,
    group_id='some-group'
    auto_offset_reset='earliest',
    session_timeout_ms=KAFKA_SESSION_TIMEOUT_MS,
    heartbeat_interval_ms=KAFKA_HEARTBEAT_INTERVAL_MS,
    max_poll_interval_ms=KAFKA_MAX_POLL_INTERVAL_MS
)
async def handle_messages(msg: KafkaMessage) -> None:
  ...

async def run_broker():
    await broker.start()
```

We starts it and broker begin to consume messages but it falls in error when message has key, but don't have value.
This error occures on the Faststream files level, that is why I am not able to handle it in my handler function.

ERROR LOG:
```
File "/home/user/.local/lib/python3.12/site-packages/faststream/broker/subscriber/usecase.py", line 402, in process_message
     raise parsing_error
   File "/home/user/.local/lib/python3.12/site-packages/faststream/broker/subscriber/usecase.py", line 343, in process_message
     message = await h.is_suitable(msg, cache)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   File "/home/user/.local/lib/python3.12/site-packages/faststream/broker/subscriber/call_item.py", line 143, in is_suitable
     message._decoded_body = cache[decoder] = cache.get(decoder) or await decoder(
                                                                    ^^^^^^^^^^^^^^
   File "/home/user/.local/lib/python3.12/site-packages/faststream/kafka/parser.py", line 53, in decode_message
     return decode_message(msg)
            ^^^^^^^^^^^^^^^^^^^
   File "/home/user/.local/lib/python3.12/site-packages/faststream/broker/message.py", line 124, in decode_message
     m = json_loads(body)
         ^^^^^^^^^^^^^^^^
   File "/usr/local/lib/python3.12/json/__init__.py", line 339, in loads
     raise TypeError(f'the JSON object must be str, bytes or bytearray, '
 TypeError: the JSON object must be str, bytes or bytearray, not NoneType
 2024-12-05 16:09:47,553 ERROR    -                                                                                                                       |                       |            - Processed
```

I have tried to handle it with custom value serializer but it doesn't help

**Comment by Lancetnik** [comment]
Can you show the published message itself? I can't reproduce the problem without it

**Comment by Korifey02** [comment]
> Can you show the published message itself? I can't reproduce the problem without it

{
	"Value": null,
	"Offset": 4209,
	"Key": "some-key",
	"Partition": 0,
	"Headers": {},
	"Timestamp": "2024-12-05T13:09:45.028Z"
}

message example from kafka-ui. As fact this message wasn't published by faststream publisher instance.

**Comment by Lancetnik** [comment]
Fixed already, wait a bit for release

**Comment by Korifey02** [comment]
> Fixed already, wait a bit for release

Thank you very much!! Can you say approximately, wait for hours, days, weeks?

**Comment by Lancetnik** [comment]
Right after PR merge

**Comment by Lancetnik** [comment]
@Korifey02 the fix already on Pypi as 0.5.33
